server/user_config.go

```diff
@@ -115,6 +119,21 @@ type UserConfig struct {
 	WebsocketCheckOrigin   bool            `mapstructure:"websocket-check-origin"`
 }
 
+// ToAllowCommandNames parse AllowCommands into a slice of CommandName
+func (u UserConfig) ToAllowCommandNames() ([]command.Name, error) {
+	var allowCommands []command.Name
+	for _, input := range strings.Split(u.AllowCommands, ",") {
```

Comment:
Should we sanitize (lowercase, trim, and remove whitespace) the input?

Reply:
Other configurations doesn't support such feature, too. e.g. `--autoplan-file-list`. I think not need it.

https://www.runatlantis.io/docs/server-configuration.html#autoplan-file-list

Reply:
Ok. Let's skip for now